README.md

```diff
@@ -35,8 +35,9 @@ We also use it to invoke all core development tools via `npm run`.
 2. [**Vite**](https://vitejs.dev/) : Vite is an asset bundler and build system for large web projects with 
 many types of resources such as typescript. Additionally, it provides excellent debugging features.
 We use it as a build system for Typescript, Asset Compressor, and Live Debugger.
+    * **Playwright** : a testing framework compatible with Vite with support for TypeScript. Very similar to Vitest but more for the UI side of things. 
```

Comment:
Is it really very similar to vitest?

Reply:
Seemed like it on first glance, I tell you